packages/uniforms/test/GraphQLBridge.js

```diff
@@ -36,22 +36,27 @@ describe('GraphQLBridge', () => {
 
     const schemaData = {
         author: {
+            initialValue: 'Jobs',
             component: 'div'
         },
         id: {
             allowedValues: [1, 2, 3],
             label: 'Post ID'
         },
         title: {
+            initialValue: 'Some Title',
             options: [
                 {label: 1, value: 'a'},
-                {label: 2, value: 'b'}
+                {label: 2, value: 'b'},
+                {label: 3, value: 'Some Title'}
             ]
         },
         votes: {
+            initialValue: '44',
```

Comment:
I tried with ...

    votes: {
      initialValue: 44,

... before committing, but the test only passed if I quoted the number?  I presume `bridge.getInitialValue('votes')` is returning a `String`.  I do not know how to fix that.